### PR TITLE
test: Can add google calendar e2e test 

### LINKED ISSUE #9371: [CAL-1885] Make E2E test: Can add Google Calendar
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1885](https://linear.app/calcom/issue/CAL-1885/make-e2e-test-can-add-google-calendar)</sub>

### PR DESCRIPTION
**Pull request by Ninad3721**
## What does this PR do?

This PR adds e2e test to check if google calendar can be added  after getting installed
Fixes https://github.com/calcom/cal.com/issues/9371

## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

npx playwright test google-calendar

## Mandatory Tasks
- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by Udit-takkar — COMMENTED** [review]
- I have pushed the changes after applying prettier formatting to the code, as the lint was failing due to the formatting issue.

- I am not able to run npx playwright test google-calendar even though i have installed @playwright/test.

Screenshot 2023-06-13 at 5 42 53 PM

**Comment by Ninad3721** [comment]
@Udit-takkar is it problem with this test only or all the test are not working ? Let me run it again on my pc qnd update you.

**Comment by Ninad3721** [comment]
@Udit-takkar I was able to run test , I think it is problem specific to your system.

**Comment by PeerRich** [comment]
@Ninad3721 can you address the lint issues?

**Comment by Ninad3721** [comment]
> @Ninad3721 can you address the lint issues?
I always lint code before pushing it but still get this error every time , No problem I will lint it again and try to edit the PR. Thank your for pointing out.

File: apps/web/playwright/goggle-calendar.e2e.ts
Comment (hariombalhara) [inline apps/web/playwright/goggle-calendar.e2e.ts:1]: Should be google-calendar.e2e.ts

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Thanks for the PR @Ninad3721 but the tests are failing in CI. Could you please take a look.
image

**Pull request closed without merging**